vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -406,7 +391,7 @@ def update_field_visibility(options={})
     update_field_read_only(options)
   end
 
-  def update_field_visibility_linked_clone(options={}, f)
+  def update_field_visibility_linked_clone(_options = {}, f)
```

Comment:
It is only called from one place so it would seem better to modify the caller to only pass the single argument the method needs.  If you do that you also need to update the method being overridden in miq_provision_redhat_workflow.rb.